### PR TITLE
chore: apply filtering to task logs

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Trial Logs currently have log message filtering applied. The standard timestamp is filtered out, and the kubernetes container id is filtered out as well. Apply the same logic to task log messages as well.

BEFORE (double timestamp):
2022-04-20 at 3 10 PM

AFTER:
2022-04-20 at 3 09 PM

UPDATED FORMAT:
Screen Shot 2022-04-21 at 3 40 58 PM

## Test Plan

- [ ] Check that the task logs don't display the timestamp twice (2nd time because it's being displayed in the message)
- [ ] Check that in kubernetes cluster task logs, the container id `[8 hex-digits]` doesn't show up

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by stoksc — APPROVED** [review]
can't approve the code, but I tested locally and it works great

File: webui/react/src/services/decoder.ts [hunk 1]
```diff
@@ -528,32 +528,36 @@ const decodeV1LogLevelToLogLevel = (level: Sdk.V1LogLevel): types.LogLevel | und
 const defaultRegex = /^\[([^\]]+)\]\s([\s\S]*)(\r|\n)$/im;
 const kubernetesRegex = /^\s*([0-9a-f]+)\s+(\[[^\]]+\])\s\|\|\s(\S+)\s([\s\S]*)(\r|\n)$/im;
 
+const filterLogMessage = (message: string): string => {
```
Comment (hamidzr) [inline webui/react/src/services/decoder.ts:531]: nit: rename? translation or formatting describes the operation better

File: webui/react/src/services/decoder.ts [hunk 2]
```diff
@@ -528,32 +528,36 @@ const decodeV1LogLevelToLogLevel = (level: Sdk.V1LogLevel): types.LogLevel | und
...
+  let filteredMessage = message.trim();
+
+  if (defaultRegex.test(message)) {
```
Comment (hamidzr) [inline webui/react/src/services/decoder.ts:534]: I'm not sure how .test is implemented but if it's just checking for matches underthe hood just doing `.match` once and storing the results should be faster considering we might be processing a lot of log lines during scroll events.

**Review by hamidzr — APPROVED** [review]
lgtm

File: webui/react/src/services/decoder.ts (hunk 2 above)
Comment (EmilyBonar) [reply webui/react/src/services/decoder.ts:534]: According to [this website](https://www.measurethat.net/Benchmarks/Show/11043/0/stringmatch-vs-regextest), `.test` is about 58% faster than `.match`.

Comment (hkang1) [reply webui/react/src/services/decoder.ts:534]: good call

**Pull request merged**